kornia/nerf/camera_utils.py

```diff
@@ -1,113 +1,16 @@
 import math
-from typing import List, Tuple
+from typing import List
 
 import torch
+from scipy.interpolate import splev, splprep
```

Comment:
i don't think we can allow this, only torch dependency

Reply:
can we implement with pytorch ?

Reply:
Well, we can implement spline interpolation, or use a Pytorch based open-source (https://github.com/balbasty/torch-interpol, e.g.). This will also have more benefits, such as GPU support, which is not available with scipy.

The whole point of this type of interpolation is to create a path that goes through the given cameras, and interpolate between. Maybe there could be a better way of doing that than spline interpolation

Reply:
this package looks interesting but not clear we want to adopt unless it us who maintain it (or we merge it with the author). Besides, was in my plan to port this: https://github.com/strasdat/Sophus/blob/23.04-beta/cpp/sophus/lie/interp/spline.h

Reply:
/cc @ducha-aiki

Reply:
Having our own spline interpolation is good :)

Reply:
I think that trying to reach out to Yaël Balbastre (torch-interpol author) with proposition to join kornia should be out first step.

Reply:
@ducha-aiki could you take that action?

Reply:
In the meantime I am going to remove the functionality that uses spline